harvester_e2e_tests/templates/provider.tf.j2

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    harvester = {
+      source  = "registry.terraform.io/harvester/harvester"
```

Comment:
to make it self-contained, shouldn't this be pointing to a local directory, relative to tests root?